packages/firebase_core/firebase_core/ios/Classes/FLTFirebaseCorePlugin.m

```diff
@@ -53,6 +53,14 @@ + (instancetype)sharedInstance {
 
 static NSMutableDictionary<NSString *, NSString *> *customAuthDomains;
 
+// Initialize static properties
+
++ (void)initialize {
+  if (self == [FLTFirebaseCorePlugin self]) {
+    customAuthDomains = [[NSMutableDictionary alloc] init];
```

Comment:
Any reason why we shouldn't just move this to line 34 where we initialise `FLTFirebaseCorePlugin`?

Reply:
It isn't correct in Objective C to initialise a static property in an instance initialiser.  Static properties need to be initialised in a static initialiser.